website/docs/intro/install/repo-yum.sh

```diff
@@ -2,21 +2,21 @@ cat >/etc/yum.repos.d/opentofu.repo <<EOF
 [opentofu]
 name=opentofu
 baseurl=https://packages.opentofu.org/opentofu/tofu/rpm_any/rpm_any/\$basearch
-repo_gpgcheck=1
-gpgcheck=0
+repo_gpgcheck=0
```

Comment:
We are disabling the repocheck because this way we won't have to deal with the double GPG key issue.